ros2doctor/ros2doctor/api/__init__.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import platform
+
+import rosdistro
+
+
+def print_platform_info():
+    """Print out platform information."""
+    # platform info
+    print('System Information')
+    print('system   : ', platform.system())
+    print('Linux distribution   : ', platform.dist())
```

Comment:
The documentation says `platform.dist()` [will be removed in python 3.8](https://docs.python.org/3/library/platform.html#platform.dist). I recommend removing this check for now

Reply:
If I use `platform.platform()` instead, it will give me this output ''Linux-5.0.0-23-generic-x86_64-with-Ubuntu-18.04-bionic'', which contains the same information as platform.dist() with additional parsing. Do you think parsing the string to keep this check is a worthy effort?

Reply:
I don't think parsing is worth the effort. The [docs say it's "human readable"](https://docs.python.org/3.7/library/platform.html#platform.platform), which sounds like it could change at any time. It might be useful to output all of `platform.platform()` anyways with a different label.